buildpipeline/pipeline.json

```diff
@@ -18,7 +18,8 @@
         {
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
-            "PB_DockerTag": "debian82_prereqs_2"
+            "PB_DockerTag": "debian82_prereqs_2",
+            "PB_TargetQueue": "Debian.82.Amd64"
           },
           "ReportingParameters": {
             "OperatingSystem": "Debian 8.2",
```

Comment:
"ReportingParameters" are only used for reporting things on MC. I don't think anything on the site would use the properties.TargetQueue (unless someone has changed the definition for that source/type combination). I can't quite tell what the Source and Type being reported through the build are based on this file... but I assume it's official/dotnet/corefx/ and test/functional/cli/, which, according the configuration files, don't differentiate on the TargetQueue value.